tracer/src/Datadog.Trace/ServiceFabric/ServiceRemotingService.cs

```diff
@@ -62,7 +62,15 @@ private static void ServiceRemotingServiceEvents_ReceiveRequest(object? sender,
                 // extract propagation context from message headers for distributed tracing
                 if (messageHeaders != null)
                 {
-                    spanContext = SpanContextPropagator.Instance.Extract(messageHeaders, &GetHeaders);
+                    spanContext = SpanContextPropagator.Instance.Extract(messageHeaders, (headers, headerName) =>
```

Comment:
```suggestion
                    spanContext = SpanContextPropagator.Instance.Extract(messageHeaders, static (headers, headerName) =>
```

Reply:
Added in 31aee89. I cannot see any changes in the codegen as expected.